ludwig/utils/data_utils.py

```diff
@@ -159,6 +159,14 @@ def wrapped_fn(*args, **kwargs):
     return wrapped_fn
 
 
+def inner_read(fp, read_fn, **kwargs):
+    if "get_preview" in kwargs:
+        chunksize = kwargs["get_preview"]
```

Comment:
This and the following line could be condensed to `chunksize = kwargs.pop("get_preview", None)`, which should return `None` rather than raise a `KeyError` if `"get_preview"` does not exist.